src/meta/client/MetaClient.cpp

```diff
@@ -1055,6 +1056,9 @@ MetaClient::regConfig(const std::vector<cpp2::ConfigItem>& items) {
 
 folly::Future<StatusOr<std::vector<cpp2::ConfigItem>>>
 MetaClient::getConfig(const cpp2::ConfigModule& module, const std::string& name) {
+    if (!configReady_) {
+        registerCfg();
```

Comment:
getConfig() used in graphd's executor, and registerCfg() is a sync call, in graphd executor should not exist sync call. Can we do` if configReady_ is not true, direct return error, like the schema?`

Reply:
Yeah, it do block graphd. The reason I add this logic is that if graphd register failed during initialization, that would take another `load_data_interval_secs`, during that time users can't show/get/update, do you think it's ok?

Reply:
yes